www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/httpserver.xml

```diff
@@ -20,6 +20,13 @@
     <label>Default Server</label>
     <type>checkbox</type>
   </field>
+  <field>
+    <id>httpserver.ssl_reject_handshake</id>
+    <label>Reject SSL Handshake</label>
+    <type>checkbox</type>
+    <help>If enabled, SSL handshakes for this server will be rejected.</help>
```

Comment:
I never use SSL in the UI - it is just wrong.

```suggestion
    <help>If enabled, TLS handshakes for this server will be rejected.</help>
```

Reply:
sorry, mimicked nginx docs